File: CS/PredefinedParametersInUriJsonSource/Form1.cs
using DevExpress.XtraReports.Wizards;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace PredefinedParametersInUriJsonSource {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        void PredefinedParametersInWizard_Click(object sender, EventArgs e) {
            var reportDesignTool = new DevExpress.XtraReports.UI.ReportDesignTool(new DevExpress.XtraReports.UI.XtraReport());
            reportDesignTool.DesignForm.DesignMdiController.AddService(typeof(IWizardCustomizationService), new MyWizardCustomizationService());
            reportDesignTool.ShowDesignerDialog();
        }
    }
}

File: VB/PredefinedParametersInUriJsonSource/Form1.vb
Imports DevExpress.XtraReports.Wizards
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports System.Windows.Forms

Namespace PredefinedParametersInUriJsonSource
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub PredefinedParametersInWizard_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
            Dim reportDesignTool = New DevExpress.XtraReports.UI.ReportDesignTool(New DevExpress.XtraReports.UI.XtraReport())
            reportDesignTool.DesignForm.DesignMdiController.AddService(GetType(IWizardCustomizationService), New MyWizardCustomizationService())
            reportDesignTool.ShowDesignerDialog()
        End Sub
    End Class
End Namespace
